libmisc/root_flag.c

```diff
@@ -79,7 +79,7 @@ static void change_root (const char* newroot)
 
 	if ('/' != newroot[0]) {
 		fprintf (log_get_logfd(),
-		         _("%s: invalid chroot path '%s'\n"),
+		         _("%s: invalid chroot path '%s', only support absolute path.\n"),
```

Comment:
Sorry, but could you change this to something like "only absolute paths are supported" or "an absolute path is required"?

Reply:
> Sorry, but could you change this to something like "only absolute paths are supported" or "an absolute path is required"?

done